### ISSUE TITLE
Sound output of Terminal Applications

### ISSUE THREAD

**Issue by eanmikale** [afi-dev/Termsteel#9]
Thank you for this awesome tool. Is there a straight-forward way to integrate the sound output of Terminal Applications running on Termsteel? Thank you again.

**Comment by afi-dev** [comment]
Yes why not ^^ 

But why do you need this particular feature?

**Comment by afi-dev** [comment]
Add to Enhancement

**Comment by kaelianbaudelet** [comment]
There is no way to get the sound output, I don't see how to do it, but if you know how I can do it I could add it. If you want I could add it from the sound depending on the interactions made

I have add this idea in Enhancement

**Comment by eanmikale** [comment]
The terminal doesn't produce the sound we've found. We needed audio players to handle audio playback, like Pyaudio. When terminal app reaches code section requiring a music file to be played, it does so. Pyglet was another library. Thus, it is not piped directly from the terminal. We needed to use gaming tech, to provide sound output from a terminal screen. Think Duke Nukem back in the day, or Number Crunchers. Hope this helps. Ean Mikale, J.D.Founder/Principal EngineerInfinite 8 Ecosystemhttps://hubs.mozilla.com/LLrGQRm/A Web3 Company ---- On Tue, 27 Dec 2022 20:48:05 -0600  ***@***.***  wrote ----
There is no way to get the sound output, I don't see how to do it, but if you know how I can do it I could add it. If you want I could add it from the sound depending on the interactions made
I have add this idea in Enhancement
